Update cloud cli docs with windows, new instructions (#4172)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/about-cloud-develop.md

---
title: About developing in dbt Cloud
id: about-cloud-develop
description: "Learn how to develop your dbt projects using dbt Cloud."
sidebar_label: "About developing in dbt Cloud"
pagination_next: "docs/cloud/cloud-cli-installation"
---

dbt Cloud offers a fast and reliable way to work on your dbt project. You can use it through an integrated development environment (IDE) in your browser or a dbt Cloud-powered command line interface (CLI):

- **[dbt Cloud CLI](/docs/cloud/cloud-cli-installation)** &mdash; Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment.

>   - Note, the dbt Cloud CLI is distinct from [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source command-line tool for dbt. Both tools are command lines that use the same dbt commands, however, the dbt Cloud CLI is tailored to run specifically on dbt Cloud's infrastructure.

Reviewer: is this clear @greg-mckeon @jtcohen6 ?

```suggestion
  - The dbt Cloud CLI and [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source project, are both command line tools that use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure and integrates those builds with all [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
```

Author: I would either leave this off, or clarify the value:

`The dbt Cloud CLI and [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source project, are both command line tools that run the same dbt commands. The dbt Cloud CLI  integrates with your dbt Cloud development environment, providing faster and cheaper builds, centralized access management and access to all dbt Cloud features.`

Reviewer: ok gotcha! i'm just trying to make sure users don't get confused with the 'cli' term we use and associate with dbt core, and the dbt cloud cli.

> - **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface (UI).

Author: This is really about not requiring setup of dbt, right?

---
title: About developing in dbt Cloud
id: about-cloud-develop
description: "Learn how to develop your dbt projects using dbt Cloud."
sidebar_label: "About developing in dbt Cloud"
pagination_next: "docs/cloud/cloud-cli-installation"
---

> dbt Cloud offers a fast and reliable way to work on your dbt project. You can use it through an integrated development environment (IDE) in your browser or a dbt Cloud-powered command line interface (CLI):

Reviewer 2: suggest comparing the two modes browser or cli. Also suggest removing "it." 
```suggestion
dbt Cloud offers a fast and reliable way to work on your dbt project. You can develop in your browser using an integrated development environment (IDE) or in a dbt Cloud-powered command line interface (CLI):
```

- **[dbt Cloud CLI](/docs/cloud/cloud-cli-installation)** &mdash; Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment.

>   - Something to note is that the dbt Cloud CLI and the open-sourced dbt Core are both command line tools that let you run your dbt projects and use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure and integrates those builds with all [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).

Reviewer 2: Suggest making this a note ☝🏻

> - **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface (UI).

Reviewer 2: Suggest not identifying "UI" unless you're going to use it later on the page or in the paragraph.
```suggestion
- **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface.
```

> This documentation section provides detailed instructions on setting up the dbt Cloud CLI and dbt Cloud IDE. To get started with dbt development, you'll need a [developer](/docs/cloud/manage-access/seats-and-users) account. For a more comprehensive setup guide, refer to our [quickstart guides](/quickstarts).

Reviewer 2: ```suggestion
The following sections provide detailed instructions on setting up the dbt Cloud CLI and dbt Cloud IDE. To get started with dbt development, you'll need a [developer](/docs/cloud/manage-access/seats-and-users) account. For a more comprehensive guide about developing in dbt, refer to our [quickstart guides](/quickstarts).
```
